src/lexer/diffeq.ll

```diff
@@ -1,12 +1,18 @@
-/*************************************************************************
- * Copyright (C) 2018-2022 Blue Brain Project
- * Copyright (C) 2018-2022 Michael Hines
```

Comment:
Should I keep that?

Reply:
I asked Michael to confirm in https://github.com/BlueBrain/nmodl/issues/1047#issuecomment-1599413047 but I am pretty sure we can remove it i.e. just use Apache 2.0.

Reply:
I have sent msg to Michael about this. I will resolve this as I don't want to block this here.